src/nvme/util.h

```diff
@@ -272,6 +273,15 @@ static inline void nvme_feature_decode_temp_threshold(__u32 value, __u16 *tmpth,
 	*thsel	= NVME_FEAT_TT_THSEL(value);
 }
 
+static inline void nvme_feature_decode_temp_threshold2(__u32 value, __u16 *tmpth,
```

Comment:
For the static inline function it is not necessary to introduce a new function. Just update the existing. We don't guarantee on source code level backwards compatibility. Just on binary interface.